x-pack/platform/plugins/shared/lens/public/datasources/form_based/to_esql.test.ts

```diff
@@ -282,7 +282,8 @@ describe('to_esql', () => {
     expect(esql).toEqual(undefined);
   });
 
-  it('should work with iana timezones that fall udner utc+0', () => {
+  // this fails now with the daylight saving time changes
+  it.skip('should work with iana timezones that fall udner utc+0', () => {
```

Comment:
Fails in main too due to the daylight saving change, now London is UTC+1. I skipped it but @elastic/kibana-visualizations take care of it on a follow up PR.

Reply:
@markov00 fixed the test in another PR, I am not skipping anything so disregard this comment